### ISSUE TITLE
Wavegrad training on Arabic data set Bug report. 

### ISSUE THREAD

**Issue by Amels404** [mozilla/TTS#701]
Hello, I have tried to run the wavegrad vocoder on Arabic dataset. 
At first everything works perfectly, until an issue emerged after 1000 epochs. 
Below you can find more information about the issue. 
Thank you for your time! 

![image](https://user-images.githubusercontent.com/71659848/116380044-825a6180-a80b-11eb-9360-a59bcfc90ab7.png)

with a warning 

![image](https://user-images.githubusercontent.com/71659848/116380659-11677980-a80c-11eb-8f86-5adaee1c0b55.png)

**Comment by erogol** [comment]
Wavegrad is quite sensitive to hyperparameters.

A couple of suggestions:
1. Try a smaller learning rate
2. Disable mixed_precision training in config.json 
3. Check your dataset if there is any specific sample causing the problem. 

BTW Mozilla TTS is not maintained anymore (at least ATM) consider moving over to :frog: [Coqui TTS](https://github.com/coqui-ai/TTS). Basically the same code but better :)

**Comment by Amels404** [comment]
Thank you very much for the suggestions, this seems helpful!  I'll  try to see what I can do!
